pkg/odo/cli/pipelines/wizard.go

```diff
@@ -90,6 +114,46 @@ func (io *WizardParameters) Complete(name string, cmd *cobra.Command, args []str
 	return nil
 }
 
+func checkBootstrapDependencies(io *WizardParameters, kubeClient kubernetes.Interface) error {
+
+	client := utility.NewClient(kubeClient)
+	log.Progressf("\nChecking dependencies\n")
+
+	sealedSpinner := log.Spinner("Checking if Sealed Secrets is installed at kube-system namespace")
+	err := client.CheckIfSealedSecretsExists(sealedSecretsNS+"s", sealedSecretsName)
+	if err != nil {
+		sealedSpinner.WarningStatus("Please install Sealed Secrets from https://github.com/bitnami-labs/sealed-secrets/releases")
+		sealedSpinner.End(false)
+	} else {
+		io.SealedSecretsService.Name = sealedSecretsName
+		io.SealedSecretsService.Namespace = sealedSecretsNS
+		sealedSpinner.End(true)
+	}
+
+	argoSpinner := log.Spinner("Checking if ArgoCD Operator is installed at argocd namespace")
+	err = client.CheckIfArgoCDExists(argoCDNS)
+	if err != nil {
```

Comment:
Do we actually require ArgoCD now?

Reply:
We add the excluded resources (taskruns and pipelineruns) to the ArgoCD operator resource. Applying the generated resources without installing the ArgoCD operator will throw an error. Hence we're checking if it's installed